.github/workflows/pytest.yml

```diff
@@ -0,0 +1,40 @@
+# Install Python dependencies and run unit tests.
+#
+# Note: These tests should run quickly; we set a timeout to
+# 15 minutes to catch possible hang-ups, should they occur.
```

Comment:
Opening this thread to remember we may need to adjust to shorten run time.

Reply:
@jmshapir addressed in 326c24c and 3ee123b

Reply:
@MosesStewart thanks! I flagged a syntax issue in a separate thread.

Can you give me a sense of the main things you did to shorten run time?

Reply:
@jmshapir for all the places where we used to be generating data and calculating the bias-corrected bandwidth, I replaced with the example data and set the `best_bandwidth_value` manually.

Reply:
@MosesStewart sounds great thanks! I'm assuming we're still checking the substantive aspects of the functions that we wanted to check, so I'll resolve. If that's not the case let me know and we can discuss.